### ISSUE TITLE
Add new DAEMON_OLD_VERSION flag to health_status_interp

### ISSUE THREAD

**Issue by anthonyeleven** [digitalocean/ceph_exporter#194]
From the Pacific re notes

```
* A new health check, DAEMON_OLD_VERSION, will warn if different
 versions of Ceph are running on daemons. It will generate a health
 error if multiple versions are detected.  This condition must exist
 for over mon_warn_older_version_delay (set to 1 week by default) in
 order for the health condition to be triggered.  This allows most
 upgrades to proceed without falsely seeing the warning.  If upgrade
 is paused for an extended time period, health mute can be used like
 this "ceph health mute DAEMON_OLD_VERSION --sticky".  In this case
 after upgrade has finished use "ceph health unmute
 DAEMON_OLD_VERSION".
```

This should be added to the interpreted health as a more-serious warning.

**Comment by baergj** [comment]
:+1: We have plans to make the codebase better suited to Octopus and Pacific, but no solid timeline on it yet. We would probably accept a patch to the Nautilus branch in the meantime for this, though.

**Comment by sosodev** [comment]
This is included in the 4.0.0 release :)
